x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_details.test.tsx

```diff
@@ -328,219 +328,6 @@ describe('rule_details', () => {
     });
   });
 
-  describe('disable/enable functionality', () => {
```

Comment:
Moved these tests into Rule and RuleStatusPanel because the RuleStatusDropdown has been moved into the area behind the loading spinner.